docs/sources/dashboards/manage-dashboards/index.md

```diff
@@ -32,59 +32,61 @@ weight: 8
 
+For more information about creating dashboards, refer to [Add and organize panels](../add-organize-panels).
+
+## Browse dashboards
+
+On the Dashboards page, you can browse and manage folders and dashboards, including:
+
+- Creating folders and dashboards
+- Move dashboards between folders
+- Delete multiple dashboards and folders
+- Navigate to a folder
+- Manage folder permissions. For more information, see [Dashboard permissions](https://grafana.com/docs/grafana/latest/administration/roles-and-permissions/#dashboard-permissions)
+
+{{% admonition type="note" %}}
+As of Grafana 10.2 there is no longer the special 'General' folder. Dashboards without a folder are now shown at the top level along side folders.
+{{% /admonition %}}
+
 ## Create a dashboard folder
 
 Folders help you organize and group dashboards, which is useful when you have many dashboards or multiple teams using the same Grafana instance.
 
-**Before you begin:**
-
-- Ensure that you have Grafana Admin or Super Admin permissions. For more information about dashboard permissions, refer to [Dashboard permissions][].
+> **Before you begin:** Ensure you have Editor or greater permission to create folders. For more information about dashboard permissions, refer to [Dashboard permissions][].
 
 **To create a dashboard folder:**
 
-1. Sign in to Grafana.
 1. Click **Dashboards** in the left-side menu.
 1. On the Dashboards page, click **New** and select **New folder** in the dropdown.
 1. Enter a unique name and click **Create**.
 
-   Do not use the name of a folder that has already been provisioned (such as "General") and avoid special characters (except underscores and hyphens).
-
 When you save a dashboard, you can either select a folder for the dashboard to be saved in or create a new folder.
 
-## Manage dashboards
-
-On the Dashboards page, you can:
-
-- create a folder
-- create a dashboard
-- move dashboards into folders
-- delete multiple dashboards
-- navigate to a folder page where you can assign folder and dashboard permissions
+**To edit the name of a folder:**
 
-### Dashboard folder page
+1. Click **Dashboards** in the left-side menu.
+1. Navigate to the folder by selecting it in the list, or searching for it.
+1. Click the pencil icon labelled **"Edit title"** in the header
 
-You can complete the following tasks on a dashboard folder page:
+The new folder name is automatically saved.
 
-- Move or delete dashboards in a folder
-- Rename a folder (available on the Settings tab)
-- Assign permissions to folders (which are inherited by the dashboards in the folder)
+### Folder permissions
 
-To navigate to the dashboard folder page, hover over the name of the folder and click **Go to folder** in the dashboard search result list or on the Dashboards page.
+You can assign permissions to a folder. Any permissions you assign are inherited by the dashboards in the folder. You can assign permissions to **Organization Roles**, **Teams**, and **Users**.
```

Comment:
```suggestion
You can assign permissions to a folder. Any permissions you assign are inherited by the dashboards in the folder. You can assign permissions to organization roles, teams, and users.
```

Reply:
Roles, Teams, and Users are discrete 'nouns' that permissions can be assigned to - It seems like these should be bolded at least?

<img width="411" alt="Screenshot 2023-10-09 at 10 59 13 am" src="https://github.com/grafana/grafana/assets/46142/44c0c4b3-2901-437b-9622-1d7b3947f3b1">

Reply:
So these are nouns, but they aren't proper nouns (people, places, product names). Also, we're not making a direct reference to the UI (e.g., Select **Role** in the drop-down list) and once we pluralize them, we can't be talking about the specific UI element anymore because the UI element isn't a plural. You'll see this pattern in the relevant docs to these areas, for example, [Team management](https://grafana.com/docs/grafana/latest/administration/team-management/) for example), where the term "team" isn't capitalized unless we're referring to it in the UI, because after that we're just talking about the generic term. 

If you really want them to be bold/caps, the sentence would need to be rewritten in such a way that you're basically referring to the UI. e.g., "You can assign permissions to various entities by selecting **Role**, **Team**, or **User** in the drop-down list." But I think that's a lot clunkier.

Reply:
That makes sense - thank you for explaining!